### PR TITLE
[FEATURE] Transaction validation.

### PR DESCRIPTION
**Pull request by Kuzirashi**

### CODE REVIEW

File: app/stores/TransactionStore.js [hunk 1]
```diff
@@ -194,32 +200,90 @@ export class TransactionStore {
...
+      gasPrice,
+      fee,
+      payment,
+      requiredDeposit,
+      isTimestamp,
+      endowment
+    );
+
+    if (!paramsValid && errors.length > 0) {
+      showNotification(PARAMS_ERROR_TO_MESSAGE_MAPPING[errors[0]]);
```
Comment (Bagaric) [inline app/stores/TransactionStore.js]: Does this also work if more errors are thrown? Seems like it only displays 1 error at a time.

Comment (Kuzirashi) [reply app/stores/TransactionStore.js]: Yes, so far it displays only the first error.

Comment (Kuzirashi) [reply app/stores/TransactionStore.js]: Actually I believe that this logic will be replaced by validation earlier. I don't think user should be able to go past first step without setting recipient address or too early date. Now you can speed click up to the end. :)

Comment (Bagaric) [reply app/stores/TransactionStore.js]: @Kuzirashi The step-skipping is one of the requirements of the DApp afaik. The user should be able to skip back and forth. I had the same thoughts as you have, but then @adibas03 and @kosecki123 reminded me about the requirements. There was a discussion around that a while ago I remember.

**Pull request merged**